### PR TITLE
Install yamllint

### PR DESCRIPTION
**Pull request by sethvargo**
This will fail until https://github.com/abcxyz/pkg/pull/132 is merged.

### CODE REVIEW

**Review by capri-xiyue — APPROVED** [review]

File: .github/workflows/check.yml [hunk 1]
```diff
@@ -25,7 +25,7 @@ jobs:
           files: |
             ${{ inputs.resource_mapping_directory }}/**/*.{yaml,yml}
       - name: 'Copy added and changed files'
-        if: ${{ steps.changed-yaml-files.outputs.any_changed == 'true' }}
+        if: '${{ steps.changed-yaml-files.outputs.any_changed == ''true'' }}'
```
Comment (sqin2019) [inline .github/workflows/check.yml:28]: @sethvargo Hi, with this, vscode IDE gives an error even though workflows do not complain, wondering what setting in the IDE can fix it.
```
Unexpected symbol: 'true'''. Located at position 30 within expression: needs.check.outputs.aod == ''true''
```

Comment (sethvargo) [reply .github/workflows/check.yml:28]: Hey @sqin2019 - hmm that feels like a buge in the editor. Which plugin is throwing the error?

Comment (sqin2019) [reply .github/workflows/check.yml:28]: Not sure, will check, so no special IDE settings are needed for this I assume.

Comment (sqin2019) [reply .github/workflows/check.yml:28]: seems like it is the github action extention https://marketplace.visualstudio.com/items?itemName=GitHub.vscode-github-actions

Comment (sethvargo) [reply .github/workflows/check.yml:28]: Got it - it feels like an extension of https://github.com/github/vscode-github-actions/issues/60? Do you mind filing an issue there?

Comment (sqin2019) [reply .github/workflows/check.yml:28]: https://github.com/github/vscode-github-actions/issues/211

**Pull request merged**